frontend/src/routes/[[lang=locale]]/candidate/(protected)/profile/+page.svelte

```diff
@@ -115,7 +131,7 @@
   const dot = '\u22C5';
 
   // map nominations into objects
-  const nominationFields = nominations?.map((nom) => {
+  const nominationFields = (nomination ? [nomination] : []).map((nom) => {
```

Comment:
This seems to be a leftover of having multiple nominations, perhaps this should just be refactored not to use an array at all?